### DISCUSSION TITLE
希望支持多吉云存储功能

### DISCUSSION THREAD

**Discussion by anzhiyu-c** [lsky-org/lsky-pro#570] [想法]
拥有完善的功能与sdk，切可以直接开启审核文件，生命周期与图像处理。
多吉云存储地址：https://console.dogecloud.com/oss
sdk文档: https://docs.dogecloud.com/oss/sdk-s3upload-js

**Comment by hanasaki-misaki** [comment]
正好在用，如果支持的话管理图片会方便不少( •̀ ω •́ )✧

**Reply by anzhiyu-c** [reply]
贼期待QWQ

**Comment by hanasaki-misaki** [comment]
> 拥有完善的功能与sdk，切可以直接开启审核文件，生命周期与图像处理。 多吉云存储地址：https://console.dogecloud.com/oss sdk文档: https://docs.dogecloud.com/oss/sdk-s3upload-js

文档里说不介入小厂的存储，估计没戏但是改改能用
参考文章：https://lolicon.team/posts/7b911d0b.html

**Reply by anzhiyu-c** [reply]
多吉不是小厂😭!
